signingscript/requirements/base.in

```diff
@@ -3,6 +3,6 @@ mar>=3.0.0
 scriptworker
 taskcluster
 mohawk>=1.0.0
-winsign>=2.2.2
+winsign==2.2.3
```

Comment:
We could add a comment about how winsign 2.2.3 is compatible with osslsigncode 2.1, and that we'll need a new winsign release for osslsigncode 2.2. Alternately we could file an issue, so we're not caught off guard the next time we upgrade our docker image base.